#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_g++-4.3.dpatch by Pierre Habouzit <madcoder@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: g++-4.3 patch

@DPATCH@
--- qgo-1.5.4-r2~/src/matrix.cpp	2004-10-30 20:49:50.000000000 +0000
+++ qgo-1.5.4-r2/src/matrix.cpp	2008-03-16 23:04:18.306414183 +0000
@@ -2,12 +2,14 @@
 * matrix.cpp
 */
 
+#include <cstdlib>
 #include "matrix.h"
-#include <stdlib.h>
 #ifndef NO_DEBUG
-#include <iostream.h>
+#include <iostream>
 #endif
 
+using namespace std;
+
 Matrix::Matrix(int s)
 : size(s)
 {
--- qgo-1.5.4-r2~/src/tree.cpp	2005-05-12 20:58:23.000000000 +0000
+++ qgo-1.5.4-r2/src/tree.cpp	2008-03-16 23:04:50.660267034 +0000
@@ -5,7 +5,7 @@
 #include "tree.h"
 #include "move.h"
 #include "qgo.h"
-#include <iostream.h>
+#include <iostream>
 #include <qptrstack.h>
 
 Tree::Tree(int board_size)
